java/code/src/com/redhat/rhn/taskomatic/core/TaskomaticDaemon.java

```diff
@@ -157,4 +225,26 @@ private void createOption(Options accum, String longopt, boolean arg,
             this.masterOptionsMap.put(longopt, option);
         }
     }
+
+    /**
+     * Registers the daemon implementation with the Tanuki wrapper
```

Comment:
Is this correct to keep mentioning `Tanuki wrapper`?

Reply:
i noticed that and started taking care of it.